app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module TwoFactorAuthCode
+  class PhoneDeliveryPresenter < TwoFactorAuthCode::GenericDeliveryPresenter
+    include TwoFactorAuthCode::Totpable
+    include TwoFactorAuthCode::Phoneable
```

Comment:
Ah, I missed it when I was searching the other PRs. It's fine if they are being used elsewhere.